Source/GRChomboCore/ChomboParameters.hpp

```diff
@@ -103,30 +67,151 @@ class ChomboParameters
                 }
             }
         }
-        pout() << "Center has been set to: ";
-        FOR1(idir) { pout() << center[idir] << " "; }
-        pout() << endl;
-
         if (nonperiodic_boundaries_exist)
         {
             // write out boundary conditions where non periodic - useful for
             // debug
             BoundaryConditions::write_boundary_conditions(boundary_params);
         }
 
```

Comment:
Suggestion - maybe we can put this in a small separate function called read_grid_params() and call it here, just to hide the detail from the main flow. I am doing the same for the boundary condition code in my update as this is also starting to clutter up read_params() too.

Reply:
Done